tests/integration/targets/cloudtrail/_config-file.json

```diff
@@ -0,0 +1,12 @@
+
+{
+"ansible_test": {
+    "environment": {
+        "ANSIBLE_DEBUG_BOTOCORE_LOGS": "True"
+    },
+    "module_defaults": null
+},
+"resource_prefix": "0c12e972",
```

Comment:
resource_prefix, tiny_prefix , and the region all come from ansible-test, these don't need to be hard-coded.